README.md

```diff
@@ -1,11 +1,48 @@
 # ALBA
+This is a Rust library of _Approximate Lower Bound Arguments_ proposed in the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/), May 2024, Eurocrypt'24 by _Pyrros Chaidos, Aggelos Kiayias, Leonid Reyzin, Anatoliy Zinovyev_.
 
-Implementation of _Approximate Lower Bound Arguments_ from the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/) published by IOG Research:
+### Introduction
+ALBA enables a prover who has a large collection of data to convince a verifier that their set includes at least a minimum number of elements that meet a specific condition, called a predicate, even though the prover only reveals a subset of the data.
+By approximating a lower bound on the prover's knowledge, ALBA makes use of a controlled gap between the size of prover's actual knowledge and the threshold of the verifier checks they know.
+This design results in highly efficient proofs, achieving nearly optimal proof sizes in both non-interactive and distributed environments.
```

Comment:
The proof sizes are not really optimal, definitely not compared to SNARKs. They are only optimal in special models like when there is a weight oracle that cannot be encoded in a circuit (e.g. a human judge). The main advantage of alba is fast proving times, especially compared to SNARKs.

Suggest replacing "optimal" with "small" and emphasizing that the prover and verifier are very fast.